src/apply.js

```diff
@@ -412,11 +414,16 @@ function setBackground(mapOrLayer, layer) {
     return _colorWithOpacity(bg, opacity);
   }
   if (typeof mapOrLayer.getTargetElement === 'function') {
+    // map object
     if (mapOrLayer.getTargetElement()) {
       updateStyle();
     }
     mapOrLayer.on(['change:resolution', 'change:target'], updateStyle);
+  } else if (mapOrLayer instanceof LayerGroup) {
+    // group object
+    // ??? could apply to first layer, but probably doesn't exist yet
```

Comment:
You could set it as property on the group (e.g. `group.set('mapbox-background', updateStyle)`. When you add the first layer to the group, pick it up from the group and set it on the layer.

Reply:
It looks like the API has `setBackground` only on vector layers (for obvious reasons?), so I'll check for the first instance of one in the group.

Reply:
I ahve this ready, but see comment to next conversation...

Reply:
Just noticed that a vector source doesn't have `setBackground` so looks like by design.